docs/api-docs/channels/channels-storefront-tutorial.md

```diff
@@ -143,55 +109,31 @@ The following protected sections are provided by BigCommerce.
 | `Notifications`       | `notifications`       | Renders channel specific notification settings |
 | [`Currencies`](#currencies-settings)          | `currencies`          | Renders channel specific currency settings     |
 
-Include protected sections in the [create channel request](/api-reference/store-management/channels/channels/createchannel) to display BigCommerce provided channel specific settings.
+Include protected sections in the [create channel menus request](/api-reference/store-management/channels/channels/postChannelMenus) to display BigCommerce-provided channel-specific settings.
 
 ```http
-POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/channels
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/channels/{{CHANNEL_ID}}/channel-menus
 X-Auth-Token: {{ACCESS_TOKEN}}
 Content-Type: application/json
 Accept: application/json
 
 {
-  "name": "Dog US",
-  ...
-  "config_meta": {
-    "app": {
-      "id": 112233,
-      "sections": [
-        {
-          "title": "Storefront Settings",
-          "query_path": "storefront_settings"
-        },
-        {
-          "title": "Domains",
-          "query_path": "domains"
-        },
-        {
-          "title": "Notifications",
-          "query_path": "notifications"
-        },
-        {
-          "title": "Currencies",
-          "query_path": "currencies"
-        }
-      ]
-    }
-  }
+  "bigcommerce_protected_app_sections": [
+    "storefront_settings",
+    "domains",
+    "notifications",
+    "currencies"
+  ]
 }
 ```
 
 Included protected sections display above custom sections.
 
 ![Protected Sections](https://storage.googleapis.com/bigcommerce-production-dev-center/images/channels/channels-sf-protected-custom-settings.png "Protected Sections")
 
-<!-- theme: warning -->
-> #### Note
-> Any content an app attempts to render to the control panel iFrame for a protected section will be overridden by the BigCommerce provided content.
-
-
 ## Storefront settings
 
-Include the `Storefront Settings` [protected section](#protected-ui-sections) in the channel's `config_meta` object to render the BigCommerce provided **Storefront Settings** tab on the channel's settings page.
+Include the `storefront_settings` [protected section](#protected-ui-sections) in the Channel Menus' `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Storefront Settings** tab on the channel's settings page.

```

Comment:
get rid of tab and provide more clarity on this. Recommended: "Include the `storefront_settings` [protected section](#protected-ui-sections) in the Channel Menus' `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Storefront Settings** menu item in the channel's navigation and corresponding settings page.